### PR TITLE
Updating the djanago storage version

### PR DESCRIPTION
**Pull request by manikarthikk**
#### What does this PR do? Please provide some context
we are updating the django storage verion.

#### Where should the reviewer start?
Right now we are unable to run the CMS migrations in our platform. after updating the django version we can run them without any errors.
#### How can this be manually tested? (brief repro steps)
Before: Error on running CMS migrations.
After: Migrations will run sucessfully.
#### What are the relevant TFS items? (list id numbers)
100043
#### Definition of done:
- [x] Title of the pull request is clear and informative
- [x] Add pull request hyperlink to relevant TFS items
- [x] For large or complex change: schedule an in-person review session
- [x] This change has appropriate test coverage

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging from a feature branch into the development branch then "flatten" or "squash" commits
1. If merging from the development branch into master (or porting changes from upstream) then use github's UI to get review feedback, but use the git command line interface to complete the actual merge.

#### Reminders AFTER merging
1. Delete the remote feature branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged from the development branch into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc). In this case, use github's UI for feedback and the git command line interface for the actual merge.

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 2 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 3 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 4 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

File: requirements/edx/base.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@ django-sekizai==0.8.2
 django-ses==0.7.1
 django-simple-history==1.6.3
 django-statici18n==1.1.5
-django-storages==1.4.1
+django-storages==1.5.2
```
Comment (sdolenc) [inline requirements/edx/base.txt]: is there a reason why we can't go to 1.6.2 or 1.6.4 ?

Comment (manikarthikk) [inline requirements/edx/base.txt]: yes, we can go to 1.6.4.
me long with @kabirkhan  verified that edx is using same 1.4.1 even in the next ginkgo release. so, we decided to test the version update with the minimal next version possible.

Note: this PR is just "[WIP]" right now. i will update the details after taking the decision about the version as a team.

Comment (sdolenc) [inline requirements/edx/base.txt]: makes sense. we don't have to go all the way to latest. i don't know what WIP means. work in progress?

Comment (manikarthikk) [inline requirements/edx/base.txt]: yes

**Pull request merged**